docs/variant_annotation.md

```diff
@@ -156,20 +153,22 @@ the double-strand break repair pathway as defined by the
 ```
 #standardSQL
 SELECT
-  reference_name, start_position, reference_bases, ALT.alt, CSQ.*
+  reference_name, start_position, reference_bases, ALT.alt, CSQ.Consequence, CSQ.Impact, CSQ.SYMBOL
```

Comment:
nit: You can break this line to be less than 80 chars. In general, I would have preferred the following style (which is also consistent with the one above):
```
SELECT reference_name, start_position, reference_bases, ALT.alt,
    CSQ.Consequence, CSQ.Impact, CSQ.SYMBOL
FROM vcf_imports_external.gnomad_genomes_chr_hg19` AS T,
    T.alternate_bases AS ALT, ALT.CSQ AS CSQ
WHERE ...
```
but up to you.

Reply:
I personally find that style to be less readable (per Google's styleguide), but broke up the lines in the middle to make them <80 chars (the "proper" way is to list each arg in one line if we break them, but I think it's overkill here :p ).